cantools/subparsers/list.py

```diff
@@ -215,11 +240,12 @@ def _do_list_nodes(can_db, args):
 
         _print_node(node)
 
-def _do_list_messages(can_db, args):
+def _do_list_messages(can_db, args, values_format_specifier):
```

Comment:
You could add type annotations.

Reply:
I added annotations to the whole subparser. fully annotating the rest of cantools is a project for future me ;)